specs/features/landing_feature.js

```diff
@@ -0,0 +1,32 @@
+var Setup = require('../features/helpers/setup');
+
+describe('Landing page', function() {
```

Comment:
Good point, but if I'm not mistaken in order to use ES6 on server side we need replace Node.js with io.js. It will be next feature. Btw it belongs to gulp tasks syntax too.